requirements/3.8/app.txt

```diff
@@ -204,6 +208,7 @@ packaging==23.2
     #   altair
     #   marshmallow
     #   matplotlib
+    #   pytest
```

Comment:
I believe something is wrong here, app.txt should not include pytest ... See also 3.10 and 3.9